doc/source/devel/make_release.rst

```diff
@@ -142,6 +121,44 @@ Release checklist
 
     python -m compileall .
 
+* Make sure you are set up to use the ``try_branch.py`` - see
+  https://github.com/nipy/nibotmi/blob/master/install.rst#trying-a-set-of-changes-on-the-buildbots
+
+* Make sure all your changes are committed or removed, because
+  ``try_branch.py`` pushes up the changes in the working tree;
+
+* Force build of your release candidate branch with the slow and big-memory
+  tests on the ``zibi`` buildslave::
+
+    try_branch.py nibabel-py2.7-osx-10.10
+
+  Check the build web-page for errors:
+
+  * https://nipy.bic.berkeley.edu/builders/nibabel-py2.7-osx-10.10
+
+* Force builds of your local branch on the win32 and amd64 binaries on
+  buildbot::
+
+    try_branch.py nibabel-bdist32-27
+    try_branch.py nibabel-bdist32-34
+    try_branch.py nibabel-bdist64-27
+
+  Check the builds completed without error on their respective web-pages:
+
+  * https://nipy.bic.berkeley.edu/builders/nibabel-bdist32-27
+  * https://nipy.bic.berkeley.edu/builders/nibabel-bdist32-34
+  * https://nipy.bic.berkeley.edu/builders/nibabel-bdist64-27
+
+  Then get the built binaries in:
+
+  * https://nipy.bic.berkeley.edu/nibabel-dist
+
+  When you've done the release to pypi, you can upload them to pypi with the
+  admin files interface.
+
+  If you are already on a Windows machine, you could have done the manual
+  command to build instead: ``python setup.py bdist_wininst``.
+
```

Comment:
Ah yes, you're right. But I personally like to avoid pushing the tag for as long as possible because it can't be reversed after.  So I suggest try_branch pushing to the buildbots after the release commit, but before pushing the release commit or the release tag.